Vagrantfile

```diff
@@ -103,14 +108,31 @@ end
 # then you can just run "vagrant --provider openstack up" and this will auto
 # configure auth
 Vagrant.configure("2") do |config|
+	clouds = YAML.load_file(ENV['HOME'] + '/.config/openstack/clouds.yaml')
```

Comment:
There's no protection against ENOENT here, but I guess the previous version would have similarly exploded if the env vars aren't set so I guess it's fine either way. An entry in the README  about using clouds.yml might be worthwhile.

Reply:
The entire thing currently explodes in a glorious manner when there is no ENV['OS_*'] defined. And it explodes in a way that really doesn't help you understand what is going wrong, as there's no indication of which file or what the line is that's exploding (basically, it pulls the variables from ENV as Nil, then the OpenStack plugin itself explodes when it's trying to parse the Nil value).

However, your comment does remind me that currently it will blow a fuse even if you're not using the OpenStack provider because it won't be able to find the file. I'm going to move these lines inside the OpenStack guard to prevent that.